strings/base_string.h

```diff
@@ -382,7 +501,15 @@ WINRT_EXPORT namespace winrt
 {
     inline bool embedded_null(hstring const& value) noexcept
     {
-        return impl::embedded_null(get_abi(value));
+        for (auto&& item : value)
```

Comment:
Consider removing our own loop and replacing with
`return std::any_of(value.begin(), value.end(), [](auto elem){ return elem == 0; });`
Or similar.

Reply:
👍